pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -56,17 +56,30 @@ func NewClient(kubeConfigPath string, requestTimeout int64) (*Client, error) {
 
 	_, err = extensionsClientset.CustomResourceDefinitions().Get("operators.kudo.dev", v1.GetOptions{})
 	if err != nil {
-		return nil, errors.WithMessage(err, "operators")
+		// timeout is not a wrappable error, timeout is an underlying issue that is NOT CRD specific, there is no value in wrapping or converting as well.
+		// best to provide the actual error for proper reporting.
+		if os.IsTimeout(err) {
+			return nil, err
+		}
+		return nil, fmt.Errorf("operators crd: %w", err)
```

Comment:
Since you're at it: I'm not sure what `"operators crd: ` adds to the error message. Maybe smth. like:
```suggestion
		return nil, fmt.Errorf("failed to fetch operators.kudo.dev CRD: %w", err)
```
?

Reply:
this request for change is not consistent with the actual change.   The message that you are wanting to change was original there.  Lets focus on the change.